media-fonts/nerd-fonts/nerd-fonts-2.1.0.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit font
+
+DESCRIPTION="Icon patched versions of popular fonts"
+HOMEPAGE="https://nerdfonts.com/"
+
+KEYWORDS="~amd64 ~arm64 ~x86"
+S="${WORKDIR}"
+
+MY_FONTS=(
+	3270
+	Agave
+	AnonymousPro
+	Arimo
+	AurulentSansMono
+	BigBlueTerminal
+	BitstreamVeraSansMono
+	CascadiaCode
+	CodeNewRoman
+	Cousine
+	DaddyTimeMono
+	DejaVuSansMono
+	DroidSansMono
+	FantasqueSansMono
+	FiraCode
+	FiraMono
+	Go-Mono
+	Gohu
+	Hack
+	Hasklig
+	HeavyData
+	Hermit
+	iA-Writer
+	IBMPlexMono
+	Inconsolata
+	InconsolataGo
+	InconsolataLGC
+	Iosevka
+	JetBrainsMono
+	Lekton
+	LiberationMono
+	Meslo
+	Monofur
+	Monoid
+	Mononoki
+	MPlus
+	Noto
+	OpenDyslexic
+	Overpass
+	ProFont
+	ProggyClean
+	RobotoMono
+	ShareTechMono
+	SourceCodePro
+	SpaceMono
+	Terminus
+	Tinos
+	Ubuntu
+	UbuntuMono
```

Comment:
Can we combine these under `ubuntu` IUSE? Just update the description in *metadata.xml* to include mono typeset too.

It's a bit more text in the ebuild, but easily doable with similar `IUSE+=` what you use already.